odoo/addons/base/models/ir_actions.py

```diff
@@ -74,6 +74,10 @@ class IrActionsActions(models.Model):
                                      ('report', 'Report')],
                                     required=True, default='action')
     binding_view_types = fields.Char(default='list,form')
+    binding_invisible = fields.Char(
+        string="Invisible attribute",
+        help="Invisible attribute, if evaluated true the action will not be shown on the action menu. This field should take a Python expression as a value",
```

Comment:
Python expression, when evaluated as true, the action isn't shown in the action menu.